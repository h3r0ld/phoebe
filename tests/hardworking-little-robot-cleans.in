start map.txt
hardworking-little-robot-random off
robot-move 0 20 10
end-turn
quit
