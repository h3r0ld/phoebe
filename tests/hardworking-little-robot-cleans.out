Hardworking-little-robot 0: New Position: Cell(20, 10)
Cell(20, 10): GameObject: Hardworking-little-robot